tests/e2e/simpleevmtest.go

```diff
@@ -210,7 +272,7 @@ func SimpleEVMTest(sc *RuntimeScenario, log *logging.Logger, conn *grpc.ClientCo
 	signer := testing.Dave.Signer
 	e := evm.NewV1(rtc)
 
-	value, err := hex.DecodeString(strings.Repeat("0", 64))
+	value, err := hex.DecodeString(strings.Repeat("0", 64-1) + "1")
```

Comment:
Maybe add a comment that this is transferring 1 base unit into the called contract?